src/clib/xtg/surf_import_irap_ascii.c

```diff
@@ -108,6 +108,10 @@ surf_import_irap_ascii(FILE *fd,
         /* convert to C order */
         x_ib2ijk(ib, &i, &j, &k, *nx, *ny, 1, 0);
         ic = x_ijk2ic(i, j, 1, *nx, *ny, 1, 0);
+        if (ic < 0) {
```

Comment:
There is a potential issue here as earlier we have: 
```
    if (mode == 0) {
              return EXIT_SUCCESS;
          }
```
leaving a lot of variables undefined.